net/qpid-proton/Portfile

```diff
@@ -58,3 +63,16 @@ variant swig description {With built-in support for SWIG so the bindings can be
                             port:swig-go \
                             port:swig-ruby
 }
+
+post-build {
```

Comment:
Since this is all only needed for testing, you might put it into a `pre-test` block instead of a `post-build` block. That way it will only get run if a user requests that the tests be run.